src/lib.rs

```diff
@@ -79,6 +79,12 @@ pub use crate::crand::ComplexDistribution;
 /// ```
 #[derive(PartialEq, Eq, Copy, Clone, Hash, Debug, Default)]
 #[repr(C)]
+#[cfg_attr(
+    feature = "rkyv",
+    derive(rkyv::Archive, rkyv::Serialize, rkyv::Deserialize,),
+    archive(as = "Complex<T::Archived>")
+)]
```

Comment:
Rust 1.31 doesn't like the compound `cfg_attr`, but it should be ok split into two, though repetitive.

Reply:
Alright, I will change that :+1: